### PR TITLE
fix: error on oci_concurrency

### LINKED ISSUE #840: OCI_CONCURRENCY Flag  Not Valid
### Environment

Device and OS: Intel Nuc, Ubuntu
App version: 0.13.1
Kubernetes distro being used: k3d
Other:

### Steps to reproduce

1. add `oci_concurrency: 3` to your `uds-config.yaml` under options and try to create the bundle

### Expected result
Bundle creates

### Actual Result
Receive a invalid config option error

### Severity/Priority
low

### Additional Context

Add any other context or screenshots about the technical debt here.

### PR DESCRIPTION
**Pull request by TristanHoladay**
## Description
Fixes the invalid config option error received when trying to use the oci_concurrency option.

## Related Issue

Fixes #840 

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Other (security config, docs update, etc)

## Checklist before merging

- [x] Test, docs, adr added or updated as needed
- [ ] [Contributor Guide Steps](https://github.com/defenseunicorns/uds-cli/blob/main/CONTRIBUTING.md) followed

### CODE REVIEW

File: src/cmd/common.go [hunk 1]
```diff
@@ -34,12 +34,13 @@ const (
 	architecture   configOption = "architecture"
 	noLogFile      configOption = "no_log_file"
 	noProgress     configOption = "no_progress"
+	ociConcurrency configOption = "oci_concurrency"
 )
```
Comment (decleaver) [inline src/cmd/common.go:38]: Looks like there is another config option missing too: `no_color`

File: src/cmd/common_test.go [hunk 2]
```diff
@@ -0,0 +1,22 @@
+package cmd
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestValidConfigOptions(t *testing.T) {
```
Comment (decleaver) [inline src/cmd/common_test.go:9]: Like the unit tests! just need to update with no_color

File: src/cmd/common.go (hunk 1 above)
Comment (TristanHoladay) [reply src/cmd/common.go:38]: nice catch, thank you!

**Review by UncleGedd — APPROVED** [review]
Tested with
```
options:
  oci_concurrency: 4
  no_color: true
```
No errors and everything works as expected

**Pull request merged**